pom.xml

```diff
@@ -73,7 +92,32 @@
 					</excludes>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.flywaydb</groupId>
+				<artifactId>flyway-maven-plugin</artifactId>
+				<version>9.22.2</version>
+				<executions>
+					<execution>
+						<id>migrate-database</id>
+						<goals>
+							<goal>migrate</goal>
+						</goals>
+					</execution>
+				</executions>
+				<configuration>
+					<url>jdbc:mysql://localhost:3306/animals_db</url>
```

Comment:
Please use properties file

Reply:
You are right, but I have trouble with carrying parameters to application.yml. I cannot connect to DB. I will try again after fixing others bugs.